packages/plugin-core/src/test/suite-integ/SyncCommand.test.ts

```diff
@@ -390,6 +396,71 @@ suite("workspace sync command", function () {
       });
     });
   });
+
+  describe("WHEN there are tracked, uncommitted changes", () => {
+    test("THEN Dendron stashes and restores the changes", (done) => {
+      runLegacyMultiWorkspaceTest({
```

Comment:
why are we using legacy vs the new style tests?

Reply:
on this note, we should do a deprecation-tagging pass on test methods (separate to this review). I get tripped up on this as well.